resources/public/api/conf/1.0/checkpoints_list.yaml

```diff
@@ -0,0 +1,49 @@
+get:
+  $ref: "./checkpoints_list.yaml"
+  summary: List checkpoints
+  description: |
+    This endpoint allows a developer to get all the checkpoints created for the vendor ID. 
+    A National Insurance number can be provided to filter the results.
+  security:
+    - User-Restricted:
+        - write:self-assessment
+  parameters:
+    - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
+    - $ref: './common/headers.yaml#/components/parameters/authorizationWriteScopeHeader'
+    - $ref: './common/queryParameters.yaml#/components/parameters/ninoCreateCheckpoint'
```

Comment:
The nino query param is marked as required, but I think it's meant to be optional:
<img width="185" alt="image" src="https://github.com/hmrc/mtd-sa-test-support-api/assets/106095575/1aa3aaba-3fc2-464f-8357-215653bb59f7">